src/python_testing/TC_RR_1_1.py

```diff
@@ -257,6 +259,25 @@ async def test_TC_RR_1_1(self):
             asserts.assert_true(attribute in basic_info[0][Clusters.BasicInformation],
                                 "Must have read back attribute %s" % (attribute.__name__))
 
+        if check_heap_watermarks:
```

Comment:
Please do this step just before `Prepare clients for subsequent fabrics`. Do not do it right in the middle here.